### PR TITLE
Home Task 16 complete

### PR DESCRIPTION
**Pull request by Goshozavr**

### CODE REVIEW

File: tests/homeTasks/homeTask16.spec.js [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+    page = await ctx.newPage();
+    await page.goto("/");
+  });
+
+  test("Open login form", async () => {
+    const loginButton = page.locator("button", {
+      hasText: "Sign in",
+    });
+
+    await loginButton.click("log in form should opens");
```
Comment (sttaran) [inline tests/homeTasks/homeTask16.spec.js]: move it to beforeEach hook

Comment (sttaran) [inline tests/homeTasks/homeTask16.spec.js]: click does not accept message

Comment (sttaran) [reply tests/homeTasks/homeTask16.spec.js]: go to page and open popup before each tests. This will make your tests independent

**Review by sttaran — CHANGES_REQUESTED** [review]

File: tests/homeTasks/homeTask16.spec.js (hunk 1 above)
Comment (Goshozavr) [reply tests/homeTasks/homeTask16.spec.js]: it`s just an expected result for this test, but ok, I`ll remove it

File: tests/homeTasks/homeTask16.spec.js [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+
+    await loginButton.click();
+  });
+
+  test.beforeEach("Open registration form", async () => {
+    const regBtn = page.locator("button", {
+      hasText: "Registration",
+    });
+
+    await regBtn.click();
```
Comment (sttaran) [inline tests/homeTasks/homeTask16.spec.js]: you do not need 2 beforeEach, keep it in one hook

**Review by sttaran — CHANGES_REQUESTED** [review]

**Pull request merged**